docs/how-to-upload-manifest.md

```diff
@@ -1,107 +1,92 @@
 # How To Push Operator Manifest(s) To App Registry
```

Comment:
How about using `artifacts` instead of manifests given that it is such an overloaded term? Alternatively we define what a manifest means in this context. This will also help us figure out whether we use singular or plural when referencing it. As it stands usage of the singular or plural form is not being done consistently across this document.

Reply:
Would `artifact` refer to data required for one operator, or each data field that's put into one file? `manifest` isn't much clearer, so I think a definition is needed either way.
Will be using singular to refer to `Operator Manifest` since the scripts push one file at a time.

Reply:
When you say `Operator Manifest` people assume CSV which is not true in this case and there in lies the problem. That needs to be called out clearly as there is a lot of confusion around this. So avoiding the term `Operator Manifest` will help.